media-libs/xvid/xvid-1.3.5.ebuild

```diff
@@ -40,6 +40,9 @@ src_prepare() {
 		-e 's/@$(RM)/$(RM)/' \
 		-e 's/@$(INSTALL)/$(INSTALL)/' \
 		-e 's/@cd/cd/' \
+		-e '/\$(libdir)\/\$(STATIC_LIB)/d' \
+		-e 's/\$(BUILD_DIR)\/\$(STATIC_LIB)//g' \
+		-e 's/info \$(STATIC_LIB)/info/g' \
```

Comment:
together with the EAPI bump, this step is rather invasive. Just revbump, drop to unstable and then I can commit it.

Reply:
done